recipes/aiida-crystal17/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "aiida-crystal17" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 03b8bf0dd46a6ef506e2894d4b403cfd97fac8a41421262dcabf72247c3d6974
+
+build:
+  number: 0
+  entry_points:
+    - mock_runcry17 = aiida_crystal17.tests.mock_runcry17:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py > 27]
```

Comment:
Is there anything preventing this from being a `noarch: python` package?

You can change the `host` and `run` dependencies to be something like `python >3.4` to avoid using selectors.

Reply:
I copied line 17 from the [aiida-core-feedstock](https://github.com/conda-forge/aiida-core-feedstock/blob/master/recipe/meta.yaml), which only supports python 2.7.
Within the next month or two they should have a v1 release finalised, which will be `noarch: python`

Reply:
Sorry I misread the selector, I should have suggested `python <3` or `python =2.7`

It looks like `aiida-core` could also be `noarch: python` using the same trick.